<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserWindow</class>
 <widget class="QDialog" name="AddUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод данных абонента </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_number">
       <property name="text">
        <string>Номер телефона звонков:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_time_inp">
       <property name="text">
        <string>Время входящих звонков: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_time_inp">
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_time_out">
       <property name="text">
        <string>Время исходящих звонков: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_time_out">
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_number"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_type">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_LEMS">
      <attribute name="title">
       <string>Юридическое лицо</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_LEMS_INN">
         <property name="text">
          <string>ИНН: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_LEMS_name">
         <property name="text">
          <string>Название: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_LEMS_name"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_LEMS_address">
         <property name="text">
          <string>Адрес:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_LEMS_addres"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_LEMS_INN">
         <property name="maxLength">
          <number>11</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_IMS">
      <attribute name="title">
       <string>Физическое лицо</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_IMS_full_name">
         <property name="text">
          <string>ФИО: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_IMS_full_name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_IMS_passport_data">
         <property name="text">
          <string>Паспортные данные: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_IMS_passport_data"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_IMS_balance">
         <property name="text">
          <string>Баланс счета: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_IMS_balance">
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_EMS">
      <attribute name="title">
       <string>Собственный сотрудник</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_EMS_full_name">
         <property name="text">
          <string>ФИО: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_EMS_full_name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_EMS_service_number">
         <property name="text">
          <string>Табельный номер: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_EMS_service_number">
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
